Document API onboarding account requirement (#2186)
MultiSafepay/docs: content/my-account/api-onboarding.md

The API also lets you create a customized onboarding process for new merchants depending on your business model, e.g. marketplaces, franchises, and crowdfunding platforms. This is particularly useful if you are onboarding multiple merchants.  

> 📘 Supported accounts
> 

> > The requests to onboard affiliated merchants via our API require a partner or primary account. For more information, see [Create account](/docs/create-account/).

Reviewer: ```suggestion
> Requests to onboard affiliated merchants via our API require a partner account API key. For more information, email your partner manager.
```
This is what we used to have:

> All affiliate onboarding requests require a partner account API key. This is not the same as a website API key. For more information, email your partner manager.
